tests/microgrid/test_microgrid_api.py

```diff
@@ -82,23 +83,39 @@ def connections(self) -> list[list[Connection]]:
         ]
         return connections
 
+    # ignore mypy: Untyped decorator makes function "components" untyped
```

Comment:
i don't see any function `components` that this seems to referring to?

Reply:
Right, this was probably a copy/paste error. I've already removed this comment